### ISSUE TITLE
Importing Data?

### ISSUE THREAD

**Issue by Allan-Cao** [thomasahle/fastchess#1]
Is it possible to import PGNs instead of using Stockfish to generate them?

**Comment by thomasahle** [comment]
That shouldn't be hard to do. They just have to be transformed to the same format make_data.py uses. 

The pgns would have to have evaluations of course. Do you have a large collection of such data? Maybe we can get it from the Leela project.

**Comment by thomasahle** [comment]
I have added a new file `proc.py` that lets you import games form pgn files.
```
python proc.py 'cclr/**/*.pgn' -test proc.test -train proc.train
fasttext supervised -input proc.train -output proc.model -t 0 -neg 0 -epoch 1
fasttext test proc.model.bin proc.test 1
python3 play_chess.py proc.model.bin -debug -selfplay
```
I trained it on four million games from CCLR and it gets some pretty great recall rates:
`R@1	0.227,
R@2	0.355,
R@3	0.445,
R@4	0.515,
R@5	0.572`.
For comparison [Leela Zero has accuracy/R@1](http://testtraining.lczero.org/) of 50-60%.

And this is with just a 100-dimensional hidden layer/embedding. Cool stuff.
